pymc/smc/sample_smc.py

```diff
@@ -56,50 +56,53 @@ def sample_smc(
 
     Parameters
     ----------
-    draws: int
+    draws : int, default 2000
         The number of samples to draw from the posterior (i.e. last stage). And also the number of
         independent chains. Defaults to 2000.
-    kernel: SMC Kernel used. Defaults to pm.smc.IMH (Independent Metropolis Hastings)
-    start: dict, or array of dict
+    kernel : class, default `pymc.smc.smc.IMH`
+        SMC_Kernel used. Defaults to :class:`pymc.smc.smc.IMH` (Independent Metropolis Hastings)
+    start : dict, or array of dict, default None
         Starting point in parameter space. It should be a list of dict with length `chains`.
         When None (default) the starting point is sampled from the prior distribution.
-    model: Model (optional if in ``with`` context)).
-    random_seed: int
-        random seed
-    chains : int
+    model : Model (optional if in ``with`` context).
+    random_seed :  int, array_like of int, RandomState or Generator, optional
```

Comment:
This is great 😄. The {} you mention are only to be used when a parameter can only take a fixed set of _values_ not of _types_. For example, one could use `int or str` to describe the types, but if only `{0, 1, "column", "row"}` are valid then instead of describing the type, we can use curly brackets to indicate only those specific values are accepted

Reply:
Thanks!